### PR TITLE
Olm 3.1.4

### PR DESCRIPTION
**Pull request by extrowerk**

### CODE REVIEW

File: app-crypt/olm/olm-3.1.4.recipe [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+HOMEPAGE="https://gitlab.matrix.org/matrix-org/olm"
+COPYRIGHT="2015-2016 OpenMarket Ltd.
+	2018-2019 New Vector Ltd."
+LICENSE="Apache v2"
+REVISION="1"
+SOURCE_URI="https://gitlab.matrix.org/matrix-org/olm/-/archive/$portVersion/olm-$portVersion.tar.bz2"
+CHECKSUM_SHA256="eff513cff8e5c7af657e800a2c2dbbdee5be19ee9ce24c5498a2528c4b5f1dc3"
+
+ARCHITECTURES="?x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="?x86_gcc2 x86"
```
Comment (Begasus) [inline app-crypt/olm/olm-3.1.4.recipe]: Builds OK for secondary architecture on gcc2h, with gcc2 it fails with https://paste.ubuntu.com/p/QR7gjgcgcD/
(Tests look OK also for secondary architecture

File: app-crypt/olm/olm-3.1.4.recipe [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+	"
+
+defineDebugInfoPackage olm$secondaryArchSuffix \
+	"$libDir"/libolm.so.$libVersion
+
+BUILD()
+{
+	mkdir -p haiku_build && cd "$_"
+
+	# Test fals to build, complains about fPIC
```
Comment (Begasus) [inline app-crypt/olm/olm-3.1.4.recipe]: Seems to work OK on gcc2h for secondary architecture, removed line 54 from the recipe, builds OK and tests run OK also?

**Comment by Begasus** [comment]
Ready to merge?

**Comment by extrowerk** [comment]
I think yes.

**Pull request merged**